jsonnet/jsonnetfile.lock.json

```diff
@@ -22,6 +22,17 @@
       "version": "5e7af669ee35af6c8beaa37e7a30d865368d75d8",
       "sum": "kyza6itqcvQHoKeJqsFjFQuMpj8RZsudC5/2vVxw5R0="
     },
+    {
+      "name": "prometheus-operator",
```

Comment:
I believe this was removed in another PR, hence generate is failing. Can you just not commit this file, should fix it I think.

Reply:
Yeah this accidentally got in, not sure why/how. Removed.